### PR TITLE
adds web app feature for all routes

### PR DESCRIPTION
**Pull request by neriyahbutler**

### CODE REVIEW

File: functions/src/Android Version/index.ts [hunk 1]
```diff
@@ -12,43 +12,52 @@ main.use(bodyParser.json());
 
 export const getAndroidVersion = functions.https.onRequest(main);
 
-app.get('/supportedAndroidVersions', (request, response) => {
-    getSupportedAndroidVersions(response);
+app.post('/supportedAndroidVersions', (request, response) => {
```
Comment (arjun-garg-google) [inline functions/src/Android Version/index.ts:15]: why have we changed these to POST methods? By REST convention these should be GETs

Comment (arjun-garg-google) [inline functions/src/Android Version/index.ts:15]: FYI you may want to get Helena's express removal merged into master and then rebase this code.

File: functions/src/Android Version/index.ts [hunk 2]
```diff
@@ -12,43 +12,52 @@ main.use(bodyParser.json());
...
+  getSupportedAndroidVersions(response);
 });
 
 function getSupportedAndroidVersions(res: any) {
```
Comment (arjun-garg-google) [inline functions/src/Android Version/index.ts:19]: this also just looks like a bunch of formatting changes, hard to tell what if anything is actually being functionally changed. Can we separate out the formatting fixes into its own PR?

Comment (arjun-garg-google) [reply functions/src/Android Version/index.ts:19]: same for all the other functions

File: functions/src/Android Version/index.ts (hunk 1 above)
Comment (arjun-garg-google) [reply functions/src/Android Version/index.ts:15]: same for all the other functions

Comment (neriyahbutler) [reply functions/src/Android Version/index.ts:15]: I can't send a body in the request if its GET, I use the body to send the user's token to check for admin privileges

File: functions/src/Android Version/index.ts (hunk 2 above)
Comment (neriyahbutler) [reply functions/src/Android Version/index.ts:19]: Sorry about that I'll fix it

File: functions/src/Android Version/index.ts (hunk 1 above)
Comment (arjun-garg-google) [reply functions/src/Android Version/index.ts:15]: Why can't we use the authorization request header to pass that? It is almost never right to discard proper REST conventions because of something like this.

Comment (neriyahbutler) [reply functions/src/Android Version/index.ts:15]: That's a good point, I'll resolve the issue

File: functions/src/bulletin/index.ts [hunk 3]
```diff
@@ -1,31 +1,20 @@
...
-
-export const getBulletin = functions.https.onRequest(main);
-
-app.get('/bulletins', (request, response) => {
-
-  const bulletinID = request.query.bulletinid;
-  const androidVersion = request.query.androidVersion;
+export const getBulletin = functions.https.onRequest((request, response) => {
+  
+  const bulletinID = request.query.bulletinidBULLETIN;
```
Comment (gwroblew) [inline functions/src/bulletin/index.ts:7]: Is the BULLETIN suffix a leftover?
